classes/tester.php

```diff
@@ -25,11 +25,10 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-require_once($CFG->dirroot . '/local/aws/sdk/aws-autoloader.php');
-
 use Aws\S3\S3Client;
 use Aws\S3\Exception\S3Exception;
 use Aws\Iam\Exception\IamException;
+use smsgateway_aws\helper;
```

Comment:
should be ` core\aws\aws_helper`

Reply:
The core\aws\aws_helper has been deprecated since M4.5

https://github.com/catalyst/moodle-fileconverter_librelambda/issues/76

https://github.com/moodle/moodle/blob/main/lib/classes/aws/aws_helper.php#L39

Reply:
I  agree that it should be a generic core one instead smsgateway_aws, but the core is deprecated.

Reply:
Let's then do what core did - copy this string to a local helper. Seems like relying on sms gateways is not good thing log term

Reply:
Not ideal either, but probably the best way in our case

Reply:
cools, that is better, I will do that

Reply:
Hi @dmitriim ,

I have updated the patch, would you please have another look.
Thanks

Reply:
I have also updated with the following removed

      extra_plugin_runners: 'moodle-plugin-ci add-plugin catalyst/moodle-local_aws'